draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -368,10 +386,12 @@
             <t>
                 If the offerer inserts the SDP 'setup' attribute with an 'actpass' or 'passive' attribute value, the
                 offerer MUST be prepared to receive a DTLS ClientHello message (if a new DTLS association
-                is established by the answerer) from the answerer before the offerer receives the SDP answer.
+                is established by the answerer) from the answerer before the offerer receives the SDP answer. In such
+                case the offerer MAY cache the ClientHello message, but the offerer MUST NOT initiate a DTLS

```

Comment:
@rshpount - totally understand what you are saying. What I have been saying is that it needs to be the applications decision what it wants to do with this unauthenticated media. Clearly some applications will discard it for the reason you mention, however, other applications feel that they need to be able to mark it as unauthenticated but still use it. One of the reason the attack you mention is harder is because the port is unknown and windows   of time for the attack is small and typically the number of connections that can be formed is rate limited but I do agree the attack you describe is possible. There are other ways to mitigate it ( like ICE ). The main point is this is the way early media works is the applications  gets to decide what to do with unauthenticated media.